### PR TITLE
Resprites Lava Lamps + Adds Random Color Variants

### PR DESCRIPTION
**Pull request by Flaborized**
## About the PR 

Resprites lava lamps and adds groovy pink and bodacious orange variants. They pick one of the three types when they spawn randomly.

unknown

https://user-images.githubusercontent.com/53125172/197369291-f899f0bf-c161-4faa-9ad0-16c8298691fe.mp4

## Why's this needed? 

Wanted to do some spriting, saw that this was on the Wanted Resprites list, so I drew it. I think this is an improvement, looks rounder, fits perspective more cleanly, uses updated palette, animation is a bit more complicated and slower (I think). I made the color variants for fun and thought they looked neat so I included them as well. Little tiny random things like that are fun.

### CODE REVIEW

**Comment by NightmareChamillian** [comment]
Sprites seem to be a bit on the noisy side, but I really like them!

File: code/obj/item/device/lights.dm [hunk 1]
```diff
@@ -440,15 +440,22 @@
...
+	icon_off = "lava_lamp-blue1"
 	w_class = W_CLASS_BULKY
 	desc = "An ancient relic from a simpler, more funky time."
 	col_r = 0.85
 	col_g = 0.45
 	col_b = 0.35
 	brightness = 0.8
+	var/lamp_color
+
+	New()
```
Comment (frawhst) [inline code/obj/item/device/lights.dm:454]:
```suggestion
	New()
		. = ..()
```
need to call parent here or a lot of things get mad and break (always need to do that in `New()`

**Review by Gannets — APPROVED** [review]
Much better

File: code/obj/item/device/lights.dm (hunk 1 above)
Comment (Flaborized) [reply code/obj/item/device/lights.dm:454]: Whoops forgot about that, thanks!

**Comment by ZeWaka** [comment]
don't think changelog is applicable?

**Review by frawhst — APPROVED** [review]
Agree that changelog isn't needed. Looks excellent though

**Comment by Flaborized** [comment]
Whoops didn't see zewakas comment until now, removed changelog!

**Comment by WALPVRGIS** [comment]
love the colours! 👀

**Pull request merged**